ReSharper.FSharp/src/FSharp.Fantomas.Protocol/src/FantomasProtocolConstants.cs

```diff
@@ -8,6 +9,7 @@ public static class FantomasProtocolConstants
     public const string PROCESS_FILENAME = "JetBrains.ReSharper.Plugins.FSharp.Fantomas.Host.dll";
     public const string PARENT_PROCESS_PID_ENV_VARIABLE = "FSHARP_FANTOMAS_PROCESS_PID";
     public static readonly FileSystemPath LogFolder = Logger.LogFolderPath / "Fantomas";
+    public static readonly NuGetVersion Fantomas5Alpha3Version = NuGetVersion.Parse("5.0.0-alpha-003");
```

Comment:
I _think_ we could make it easier by only checking `5.0` version. It would break the first two alpha versions but I don't expect anyone to be using them when 2022.2 is released, and not having to deal with particular alpha version could make it a bit simpler perhaps.

Reply:
I don't mind looking at this specific version much, but I'd probably would add the specific alpha check myself. @DedSec256 What do you think?

Reply:
Yes, that makes sense. Let's only check 5.0 version.